doc/guide/Guide_CLIMADA_Tutorial.ipynb

```diff
@@ -85,8 +88,8 @@
     "  - keep the size of data you use as examples in the tutorial in mind\n",
     "  - we aim for computational efficiency\n",
     "  - a lean, well-organized, concise notebook is more informative than a long, messy all-encompassing one.\n",
-    "\n",
     "- follow the general CLIMADA naming convention for the notebook. For example: \"climada_hazard_TropCyclone.ipynb\"\n",
+    "\n",
     "![image-2.png](attachment:image-2.png)"
```

Comment:
It is not part of this pull request, but why do we have this image here?

Reply:
Good question, I do not really know.. shall I remove it ? I added a space just because it was attached to the text.

Reply:
Yes, let us remove it.